travis/test.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+if [ "$ARCH" == "" ]; then
+    echo 'Error: $ARCH is not set'
+    exit 1
+fi
+
+if [ "$ARCH" != "x86_64" ]; then
+    echo 'NOTICE: $ARCH is not supported yet'
```

Comment:
Why would any architecture not be supported? The tests aren't architecture dependent.

Reply:
You need to setup cross compilation stuff in order to support other architectures. This implies a considerable amount of work if you take into account that the Qt build we are using is not quite crosscompilation friendly.
Also, the results will be quite similar.